_script/check-samples

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#
+# Copyright 2020, TeamDev. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Redistribution and use in source and/or binary forms, with or without
+# modification, must retain the above copyright notice and the following
+# disclaimer.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+# A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+# OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+
+# Load environment variables for the case of execution from the Gradle build if they exist.
+BASH_PROFILE=~/.bash_profile
+BASH_RC=~/.bashrc
+
+if [ -f ${BASH_PROFILE} ]; then
+  echo "Using user's local bash profile."
+  . ${BASH_PROFILE}
+elif [ -f ${BASH_RC} ]; then
+  echo "Using user's local bash runtime configuration."
+  . ${BASH_RC}
+fi
+
+# Navigate to the folder with samples.
+cd _code
+
+# Check code samples.
+./embed-code-macos -config-path="config-v1.yml" -mode="check"
```

Comment:
Let's mention in the `README.md` that there is a prerequisite for the scripting part to work: having an ARM-based Mac.